build.rs

```diff
@@ -67,9 +67,145 @@
 use std::env;
 use std::path::{Path, PathBuf};
 
-
 const LIB_NAME: &'static str = "ring";
 
+const RING_SRC: &'static [&'static str] = &["crypto/aes/aes.c",
```

Comment:
I can't figure out how to make rustfmt do something reasonable with this. Let's format all of these variables containing lists of files like `RING_X86_64_SRC` and add `#[cfg_attr(rustfmt, rustfmt_skip)]` to each of them.